web/src/emoji_picker.ts

```diff
@@ -169,7 +182,7 @@ export function rebuild_catalog(): void {
 
     const categories = EMOJI_CATEGORIES.filter((category) => catalog.has(category.name));
     complete_emoji_catalog = categories.map((category) => ({
-        name: category.name,
+        name: category.translated,
```

Comment:
Maybe this is related to the error you see below. I think internally we want to keep using `name` but only when displaying the category in popover, we want to use the `translated` string.